uniswap_swap_detection/src/constant.ts

```diff
@@ -0,0 +1,35 @@
+import { ethers, getEthersProvider } from "forta-agent";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const alchemyApiKey = process.env.ALCHEMY_API_KEY;
+export const UNISWAP_ADDRESS = "0x1F98431c8aD98523631AE4a59f267346ea31F984";
+export const SWAP_EVENT =
+  "event Swap(address indexed sender,address indexed recipient,int256 amount0,int256 amount1,uint160 sqrtPriceX96,uint128 liquidity,int24 tick)";
+
+export const CREATED_POOL =
+  "event PoolCreated(address indexed token0,address indexed token1,uint24 indexed fee,int24 tickSpacing,address pool)";
+export const FUNCTION_EXACT =
+  " function exactInputSingle(address,address,uint24,address,uint256,uint256,uint256,uint160)";
+
+export const SWAP_ROUTER = "0xE592427A0AEce92De3Edee1F18E0157C05861564";
+export const PROVIDER = new ethers.providers.JsonRpcProvider(
+  `https://polygon-rpc.com`
+);
+export const POOL_INIT_CODE_HASH =
```

Comment:
```suggestion
export const UNISWAP_POOL_INIT_CODE_HASH =
```
Similar suggestion on detail.